### PR TITLE
Add HORIZONTAL BarChart

### PR DESCRIPTION
**Pull request by rickimaru18**
# 📲 What #

* Implemented HORIOZNTAL and HORIZONTAL_STACKED BarChart.

# 🤔 Why #

* New feature

# 🛠 How #

* Add implementation of "drawHorizontalBars" and "drawHorizontalStackedBars" functions.
* Add HORIZONTAL and HORIZONTAL_STACKED BarCharts composable preview.

# ✅ Acceptance criteria #

- [x] Composable preview added
- [x] Added UI Tests
- [x] Passing Unit Tests
- [x] Passing UI Tests

# ⏰ TODO #

* None

# 🔗 References #

* URLS or Images or Ticket links.
image
image
image

### CODE REVIEW

File: kwarchart/src/main/java/com/kwarchart/android/Chart.kt [hunk 1]
```diff
@@ -273,12 +273,12 @@ fun DrawScope.drawGrids(
...
+    yAxisEndPadding: Float = 0f,
     showHorizontalLines: Boolean = true,
     showVerticalLines: Boolean = true,
 ) {
-    val hGap = size.height / count
+    val hGap = (size.height - yAxisEndPadding) / count
     val vGap = (size.width - xAxisEndPadding) / count
-
     val startPoint = origin()
 //    val pathEffect = PathEffect.dashPathEffect(intervals = floatArrayOf(10f, 10f))
```
Comment (rexendy) [inline kwarchart/src/main/java/com/kwarchart/android/Chart.kt:283]: pls remove if not used

File: kwarchart/src/main/java/com/kwarchart/android/Chart.kt [hunk 2]
```diff
@@ -339,37 +341,37 @@ fun <T> DrawScope.drawAxes(
...
+        yTextPaint.color = 0xff000000.toInt()
 
-        val keyTextPaint = Paint()
-        keyTextPaint.textAlign = Paint.Align.CENTER
-        keyTextPaint.textSize = AXIS_VALUES_FONT_SIZE
-        keyTextPaint.color = 0xff000000.toInt()
+        val xTextPaint = Paint()
+        xTextPaint.textAlign = Paint.Align.CENTER
+        xTextPaint.textSize = AXIS_VALUES_FONT_SIZE
+        xTextPaint.color = 0xff000000.toInt()
```
Comment (rexendy) [inline kwarchart/src/main/java/com/kwarchart/android/Chart.kt]: pls create a 2 private method that returns Paint() for x and y text paint

**Review by rexendy — CHANGES_REQUESTED** [review]

File: kwarchart/src/main/java/com/kwarchart/android/Chart.kt (hunk 1 above)
Comment (rickimaru18) [reply kwarchart/src/main/java/com/kwarchart/android/Chart.kt:283]: Will be used in the future. 
v0.0.2: https://docs.google.com/spreadsheets/d/1dGGRSZ9Z4gJ-M44rUmFNp8CXjK8dtgqxttfLj8FG69M/edit#gid=0

Comment (rexendy) [reply kwarchart/src/main/java/com/kwarchart/android/Chart.kt:283]: noted @rickimaru18 

thank you

**Review by rexendy — APPROVED** [review]

**Comment by rexendy** [comment]
@rickimaru18 pls merge it to develop branch

thank you

**Pull request closed without merging**